code/src/components/ProgressBar.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+
+const ProgressBar = ({questionNum}) => {
+    return <footer className='progress-bar' >
+    <label htmlFor='progress'> {questionNum === 6 ? 'completed' : `question ${questionNum}/5`} </label>
```

Comment:
Like how you did this progress bar with a few lines of code. I will learn this from you!